### DISCUSSION TITLE
Why is it not possible to give a component a className?

### DISCUSSION THREAD

**Discussion by bitflak** [adobe/react-spectrum#1897] [General]
I was just wondering why it is not possible to give a component a className? I do for example like to give my components a descriptive className just for easier debugging. So I would like to know, what was the reasoning behind this decision?

**Comment by snowystinger** [comment]
I believe this section of our documentation will answer that question for you https://react-spectrum.adobe.com/react-spectrum/styling.html#escape-hatches
let us know if you have more questions

**Reply by bitflak** [reply]
Oh I must have overlooked it. So tha's why. Often I have the need for a className just for debugging reasons and not for styling. So nice to have this escape hatch. Thanks for the answer.
